org.osgi.test.cases.typedevent/bnd.bnd

```diff
@@ -20,3 +21,7 @@ Import-Package: ${-signaturetest}, *
 -runproperties = ${runproperties}
 
 -signaturetest                      = org.osgi.service.typedevent
+
+#for mockito
+-runproperties: \
```

Comment:
This should be done in osgi.tck.junit-platform.bnd using `-runproperties.osgi.tck`.

Reply:
moved this. thank you